plugins/org.eclipse.fordiac.ide.typemanagement/plugin.xml

```diff
@@ -115,19 +126,19 @@
    <extension point="org.eclipse.ui.commands">
     	<command
         	id="org.eclipse.fordiac.ide.typemanagement.openType"
-        	name="Open Type">
+        	name="%command.Open Type">
```

Comment:
I believe you need to remove the spaces?

Reply:
yes you are totally right. done.